Formula/sleuthkit.rb

```diff
@@ -11,19 +11,19 @@ class Sleuthkit < Formula
     sha256 "69843bf2119ca3851d07b79e5cde093eb9ac4ace1d3237a54e91c62212fdc680" => :yosemite
   end
 
-  conflicts_with "irods", :because => "both install `ils`"
-
   option "with-jni", "Build Sleuthkit with JNI bindings"
   option "with-debug", "Build debug version"
 
+  depends_on "afflib" => :optional
+  depends_on "libewf" => :optional
+
+  conflicts_with "irods", :because => "both install `ils`"
```

Comment:
This hasn't quite ended up in the right place. I presume `rubocop` tripped over there being nested `if build.with?` deps below.